src/edatetime.erl

```diff
@@ -18,13 +18,13 @@
 -export([iso8601/1, iso8601_basic/1]).
 -export([tomorrow/1, yesterday/1]).
 
--type timestamp() :: pos_integer().
+-type timestamp() :: integer().
```

Comment:
non_neg_integer() is 0..N while integer() is -1 ... N. Can the timestamp ever be negative? 0 is not negative in this context.